### DISCUSSION TITLE
Mobile: Metamask not available under Connect with Wallet options

### DISCUSSION THREAD

**Discussion by gprelatto** [Web3Auth/Web3Auth#1135] [Help]
Hey! I am having a strange issue (because it was working before) with web3auth on mobile devices using iOS.
When a user tries to connect their wallet, although they have metamask installed on the phone, they dont see it as an option.

![image](https://user-images.githubusercontent.com/52168771/209673443-9b2d93e1-0afd-4a30-8cb5-1190c7a75d81.png)

Any idea about why is this issue? I updated the sdk to the latest version, but still not working.

Thanks!

**Comment by aje247** [comment]
Bump!

**Comment by chaitanyapotti** [comment]
This should be fixed in the latest v4 release. 
For desktop, metamask will only be visible if the extension is installed.
For mobile, it will always show up

**Reply by aje247** [reply]
Experiencing this issue intermittently with v4 too

**Reply by softdevx15** [reply]
I am getting the same issue on v7.2.0 as well
